### PR TITLE
verify and login pages

### PR DESCRIPTION
**Pull request by BhuvanSharma0711**
Logo transparent is not transparent will solve that is later commits checked build and used a test gmail to send verification code

### CODE REVIEW

**Comment by Himasnhu-AT** [comment]
@BhuvanSharma0711 keep the login and register page the same. Like, keep UI the same. Also, when we click on login, redirect to verify, then dashboard. 

When we click on login, verify the details, and their format. take this as reference: https://github.com/OpenEduHq/WebClient-OpenEdu/blob/main/src/app/(auth)/signin/page.tsx

File: apps/server/backend/src/handlers/email.global.ts
Comment (Himasnhu-AT) [inline apps/server/backend/src/handlers/email.global.ts:1]: Don't require dotenv package

File: apps/web/src/app/verify/page.tsx
Comment (Himasnhu-AT) [inline apps/web/src/app/verify/page.tsx:1]: keep all these page, related to auth under `(auth)/` dir, for example `app/(auth)/verify`

same for login, register, forgot_password

**Review by Himasnhu-AT — CHANGES_REQUESTED** [review]
Also complete the comment above

**Comment by BhuvanSharma0711** [comment]
so when we click on register we do not go to verify instead we go on sign in and then when we click on sign in then we go to verify?? Also I think there is no logic in backend to generate and send verification code when we sign in

**Comment by Himasnhu-AT** [comment]
> so when we click on register we do not go to verify instead we go on sign in and then when we click on sign in then we go to verify?? 

Not like this. For now, as backend isn't deployed, redirect user to verify page when they login or signup, after that redirect to dashboard

> Also I think there is no logic in backend to generate and send verification code when we sign in

The logic is implemented, code is sent via email (using nodemailer)

**Comment by BhuvanSharma0711** [comment]
Changes done.

**Review by Himasnhu-AT — APPROVED** [review]

**Pull request merged**